workspaces/orchestrator/plugins/orchestrator-form-react/src/utils/generateUiSchema.ts

```diff
@@ -110,9 +111,19 @@ function extractUiSchema(mixedSchema: JSONSchema7): UiSchema<JsonObject> {
     } else {
       processLeafSchema(curSchema, path);
     }
+    processOrder(curSchema, path);
```

Comment:
it's not under the schema properties? shouldn't it be there? if it were it would be handled IIUC

Reply:
OK I got it, nvm